GTG/gtk/browser/taskbrowser.ui

```diff
@@ -452,7 +452,7 @@
                           </object>
                           <packing>
                             <property name="name">work_view</property>
-                            <property name="title">Workview</property>
+                            <property name="title">Next Up</property>
```

Comment:
Wouldn't "Actionable tasks" be clearer?

And this could also benefit from a (translateable) GtkTooltip that says that this view only shows tasks that have no subtasks/dependencies and that are not set to begin in the future or associated with a tag that is forbidden to be shown in this view (I guess this is why it was called workview; it allowed you to exclude certain tags from the work view...)

Reply:
Actionable could work. The only problem I see is that it's GTD lingo. I know this is what GTG is based on, but using domain-specific words set a bar of entry for users. I think most new users would wonder what actionable means, and what's the difference with active/open.

> this could also benefit from a (translateable) GtkTooltip

Yeah, we should add tooltips to all of them.

> (I guess this is why it was called workview; it allowed you to exclude certain tags from the work view...)

I thought it was called work view because it was a button in the toolbar, and the name made more sense in that context :thinking: